examples/30_extended/tasks_tutorial.py

```diff
@@ -30,7 +30,7 @@
 #
 # We will start by simply listing only *supervised classification* tasks:
 
-tasks = openml.tasks.list_tasks(task_type_id=1)
+tasks = openml.tasks.list_tasks(task_type_id=tasktypes.SUPERVISED_CLASSIFICATION)
```

Comment:
`tasktypes` is not defined here. Add an import statement: `from openml.tasks import TaskType`, then just used it directly (`TaskType.SUPERVISED_CLASSIFICATION`).